bokeh/_glyph_functions.py

```diff
@@ -22,20 +23,14 @@ def func(plot, *args, **kwargs):
             datasource = ColumnDataSource()
         else:
             datasource = source
-
-        legend_name = kwargs.pop("legend", None)
-
+        
         if not isinstance(plot, Plot):
             raise ValueError("expected plot object for first argument")
-
+        
         # TODO (bev) this seems like it should be here but invalid kwargs
         # currently get through (see also below)
         # plot.update(**kwargs)
```

Comment:
These three lines can be removed entirely. They date back to when there were "glyph functions" that also produced plots themselves.